docker/package/scripts/tezos-node-start

```diff
@@ -41,3 +41,7 @@ if [[ -z "$CERT_PATH" || -z "$KEY_PATH" ]]; then
 else
     exec "$node" run "${node_run_args[@]}" --rpc-tls="$CERT_PATH","$KEY_PATH"
 fi
+
+# Waiting till node start to respond
+
+while ! curl "$NODE_RPC_ADDR" &> /dev/null; do sleep 1; done
```

Comment:
Seems like commands that are supposed to be executed after `exec` aren't actually performed since `exec` replace the current process with the provided:
```
$ cat test.sh 
#!/usr/bin/env bash

exec echo "kek"

echo "mda"
$ ./test.sh
kek
```

Reply:
Why do we use exec here?

Reply:
I don't remember, to be honest. `baker` and `accuser` services use `<...> &` and `Type=forking` instead. Maybe we can use it in the node service as well

Reply:
Actually responding is needed for baker and not (crucial) for node, so it makes sense to wait in baker